src/Core/Components/Badge/FluentBadge.razor

```diff
@@ -1,37 +1,30 @@
 ﻿@namespace Microsoft.FluentUI.AspNetCore.Components
 @using Microsoft.FluentUI.AspNetCore.Components.Extensions
 @inherits FluentComponentBase
-<fluent-badge id="@Id"
-              class="@ClassValue"
-              style="@StyleValue"
-              appearance="@Appearance.ToAttributeValue()"
-              color="@Color.ToAttributeValue()"
-              shape="@Shape.ToAttributeValue()"
-              size="@Size.ToAttributeValue()"
-              @attributes="@AdditionalAttributes">
-    @if (IconStart != null)
-    {
-        if (ChildContent is null)
-        {
-            <FluentIcon Value="@IconStart" Color="Components.Color.Custom" CustomColor="@GetIconColor()" aria-label="@IconLabel" />
-        }
-        else
-        {
-            <FluentIcon Value="@IconStart" Slot="start" Color="Components.Color.Custom" CustomColor="@GetIconColor()" aria-label="@IconLabel" />
-        }
-    }
-    @ChildContent
-    @if (IconEnd != null)
-    {
 
-        if (ChildContent is null)
+<AddTag Name="span" TagWhen="@(() => _isAttached)" style="position: relative; display: inline-block;">
```

Comment:
You could simplify this line, creating a new "tag". So the style can be included in the .css file :-)

```suggestion
<AddTag Name="fluent-badge-container" TagWhen="@(() => _isAttached)">
```

```css
fluent-badge-container {
  position: relative;
  display: inline-block;
}
```